jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -908,11 +909,48 @@ public void testGetCreationTime_useCurrentTimestamp() throws InvalidCreationTime
 
   @Test
   public void testGetCreationTime_isoDateTimeValue() throws InvalidCreationTimeException {
-    Instant time =
-        PluginConfigurationProcessor.getCreationTime(
-            "2011-12-03T10:15:30+09:00", projectProperties);
     Instant expected = DateTimeFormatter.ISO_DATE_TIME.parse("2011-12-03T01:15:30Z", Instant::from);
-    Assert.assertEquals(expected, time);
+    List<String> validTimeStamps =
+        ImmutableList.of(
+            "2011-12-03T10:15:30+09:00",
+            "2011-12-03T10:15:30+09:00[Asia/Tokyo]",
+            "2011-12-02T16:15:30-09:00",
+            "2011-12-03T10:15:30+0900",
+            "2011-12-02T16:15:30-0900",
+            "2011-12-03T10:15:30+09",
+            "2011-12-02T16:15:30-09",
+            "2011-12-03T01:15:30Z");
+    for (String timeString : validTimeStamps) {
+      Instant time = PluginConfigurationProcessor.getCreationTime(timeString, projectProperties);
+      Assert.assertEquals("for " + timeString, expected, time);
+    }
+  }
+
+  @Test
+  public void testGetCreationTime_isoDateTimeValueTimeZoneRegionOnlyAllowedForMostStrict8601Mode() {
+    List<String> invalidTimeStamps =
+        ImmutableList.of(
+            "2011-12-03T01:15:30+0900[Asia/Tokyo]", "2011-12-03T01:15:30+09[Asia/Tokyo]");
+    for (String timeString : invalidTimeStamps) {
+      try {
+        PluginConfigurationProcessor.getCreationTime(timeString, projectProperties);
+        Assert.fail(
+            "creationTime should fail if region specified when zone not in HH:MM mode - "
+                + timeString);
+      } catch (InvalidCreationTimeException ex) {
+        // pass
+      }
+    }
+  }
+
+  @Test
+  public void testGetCreationTime_isoDateTimeValueRequiresTimeZone() {
+    try {
+      PluginConfigurationProcessor.getCreationTime("2011-12-03T01:15:30", projectProperties);
+      Assert.fail("creationTime should fail if timezone not specified");
```

Comment:
Looks like the built-in [`ISO_DATE_TIME`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME) allows this? We go against it?

> The ISO-like date-time formatter that formats or parses a date-time with the offset and zone if available, such as `'2011-12-03T10:15:30'`,

Reply:
Yeah I think I misunderstood what was going on. The real issue is that
`Instance.from(DateTimeFormatter.ISO_DATE_TIME("2011-12-03T01:15:30"))` throws an exception

Reply:
It appears to generate a valid TemporalAccessor intermediary, but that value cannot be converted to Instant.

Reply:
Aha, so `Instant.from` fails because the parsed time has no timezone info?

Reply:
right